packages/stryker/stryker.conf.js

```diff
@@ -15,7 +15,7 @@ module.exports = function (config) {
         '!test/**/*.d.ts'
       ],
       symlinkNodeModules: false,
-      mutate: ['src/**/*.ts'],
+      mutate: ['src/Stryker.ts'],
```

Comment:
Do we want this when we merge with master?

Reply:
Indeed no. I'll reset it. Was testing with it. Good call.